vesc_driver/CMakeLists.txt

```diff
@@ -1,19 +1,19 @@
 cmake_minimum_required(VERSION 3.5)
 project(vesc_driver)
 
-# Set minimum C++ standard to C++14
-if(NOT "${CMAKE_CXX_STANDARD_COMPUTED_DEFAULT}")
-  message(STATUS "Changing CXX_STANDARD from C++98 to C++14")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
-elseif("${CMAKE_CXX_STANDARD_COMPUTED_DEFAULT}" STREQUAL "98")
-  message(STATUS "Changing CXX_STANDARD from C++98 to C++14")
-  set(CMAKE_CXX_STANDARD 14)
-endif()
-
+# Set minimum C++ standard to C++17
+#if(NOT "${CMAKE_CXX_STANDARD_COMPUTED_DEFAULT}")
+#  message(STATUS "Changing CXX_STANDARD from C++98 to C++17")
+#  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
+#elseif("${CMAKE_CXX_STANDARD_COMPUTED_DEFAULT}" STREQUAL "98")
+#  message(STATUS "Changing CXX_STANDARD from C++98 to C++17")
+#  set(CMAKE_CXX_STANDARD 17)
+#endif()
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
This should use the block above that is currently commented out instead.

Reply:
Hi Josh we will uncomment the above portion but we need this line to force set C++17 unless there is a better way of doing it. Removing it will cause the compiler to fall back to C++14 which is what other packages are using.

Reply:
Please set it to C++14 and change the references to `std::optional` to `std::experimental::optional`. This should avoid the need for C++17 and be compatible with most compilers.

Reply:
This still hasn't been fixed.